test/mss/mss_player_integration.js

```diff
@@ -94,7 +94,7 @@ describe('MSS Player', () => {
   it('MSS VoD PlayReady', async () => {
     const support = await shaka.media.DrmEngine.probeSupport();
     if (!support['com.microsoft.playready']) {
-      return;
+      pending('PlayReady is not supported by the platform.');
```

Comment:
There are also two tests in `test/media/media_source_engine_integration.js` that check for support, and return on a failure. `extracts ID3 metadata from AAC` and `extracts ID3 metadata from AAC when transmuxing`. Should those also be updated to use `pending`?

Reply:
Yes, we would have to update it, I am going to create another PR for those cases.

Reply:
Got it.